src/toolboxes/background/Canvas.tsx

```diff
@@ -17,11 +17,14 @@ interface Props extends BaseProps {
 export class BackgroundCanvasClass extends Component<Props> {
   private baseCanvas: BaseCanvas;
 
-  private image: HTMLImageElement | HTMLCanvasElement;
+  private image: HTMLImageElement | ImageBitmap;
```

Comment:
Does this still need HTMLImageElement?

Reply:
Yes, for now. We use it to display the default image from public. We are planning to get rid of this in the next PR.